app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -6,7 +6,7 @@
     <div class="field">
       <label class="required">{{t 'Choose country'}}</label>
       <UiDropdown @class="search selection" @selected={{this.tax.country}} @forceSelection={{false}} @fullTextSearch={{true}}>
-        <Input @type="hidden" @autocomplete="no" @id="tax_country" @value={{this.tax.country}} />
+        <Input @type="hidden" @autocomplete="some-random-string" @id="tax_country" @value={{this.tax.country}} />
```

Comment:
Instead of `some-random-string`, try country

Reply:
It is also working by doing @autocomplete = "tax-country"  instead of using "some-random-string". Should I do that in all others also?

Reply:
Yes

Reply:
Done sir, please check now.